edge/hoymilesapi.py

```diff
@@ -361,8 +361,9 @@ def request_solar_data(self):
         )
 
         solar = self.send_payload(GET_DATA_API, header, payload)
-        if "status" in solar.keys():
```

Comment:
what was wrong with this line? If there was an issue with type of return data it should be fixed in proper place.

Reply:
Sorry, I should have started at the beginning. I opened the issue #164  about this problem.

Reply:
No, I just want to point that you fix issue in incorrect line of code. Please see line 227

Reply:
thx